modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -79,113 +78,187 @@ struct PortCfg {
         , std::tuple_size<typename Net::InArgs>::value >;
 };
 
+/**
+ * Contains description of inference parameters and kit of functions that
+ * fill this parameters.
+ */
```

Comment:
Most of the above IE comments apply here as well

Reply:
+- Applied